recipes/libutf8proc/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "libutf8proc" %}
+{% set version = "2.1.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/JuliaLang/utf8proc
+  git_tag: v{{ version }}
+  patches:
+    - 0001-CMake-Use-target_compile_definitions-to-avoid-affect.patch
+    - 0002-Static-library-support-improvements-123.patch
+    - 0003-Change-name-of-static-library-if-building-with-msvc.patch
+    - 0004-Add-install-target-for-cmake.patch
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make   # [unix]
+    - cmake  # [win]
+
+test:
+  commands:
+    - test -f "${PREFIX}/lib/libutf8proc${SHLIB_EXT}"  # [unix]
+    - test -f "${PREFIX}/lib/libutf8proc.a"            # [unix]
+    - test -f "${PREFIX}/include/utf8proc.h"           # [unix]
+
+    - if not exist "%LIBRARY_INC%\utf8proc.h"    exit 1         # [win]
+    - if not exist "%LIBRARY_BIN%\utf8proc.dll"  exit 1         # [win]
+    - if not exist "%LIBRARY_LIB%\utf8proc.lib"  exit 1         # [win]
+    - if not exist "%LIBRARY_LIB%\utf8proc_static.lib"  exit 1  # [win]
+
+about:
+  home: http://julialang.org/utf8proc/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
+  summary: a clean C library for processing UTF-8 Unicode data
+  description: |
+    utf8proc is a small, clean C library that provides Unicode normalization,
+    case-folding, and other operations for data in the UTF-8 encoding
+  doc_url: https://julialang.org/utf8proc/doc
+  dev_url: https://github.com/JuliaLang/utf8proc
+
+extra:
+  recipe-maintainers:
+    - jakirkham
+    - mingwandroid
+    - nehaljwani
```

Comment:
Went ahead you, @mingwandroid and @nehaljwani, as maintainers as it looks like you authored the package in AnacondaRecipes. Please let me know if that is ok.

Reply:
Thoughts @mingwandroid and @nehaljwani?

Reply:
Fine by me.